src/main/java/com/capitalone/dashboard/collector/SonarClient.java

```diff
@@ -19,7 +20,7 @@
      */
     void setServerCredentials(String username, String password, String token);
     List<SonarProject> getProjects(String instanceUrl);
-    CodeQuality currentCodeQuality(SonarProject project);
+    CodeQuality currentCodeQuality(SonarProject project) throws HttpClientErrorException, ParseException;
```

Comment:
@Courtneyp123  why is ParseException cascaded? Either in client or collectortask, catch block is logging exception.